Base/usr/share/man/man1/nl.md

```diff
@@ -1,11 +1,11 @@
 ## Name
 
-nl
+nl - number lines of files
 
 ## Synopsis
 
 ```sh
-$ nl [--body-numbering style] [--increment number] [--separator string] [--startnum number] [--width number] [file...]
```

Comment:
Why remove the list of options?

Reply:
At first I wanted to have it look like other man pages.
But after thinking a bit, IMO having options in the synopsis only makes sense to inform about potential clashes between options (i.e. a `|`, meaning two options can't be used altogether). Otherwise it doesn't have any benefit over the `Options` section.

Reply:
(thus I only removed the list of options from the synopsis in a few man pages)